tests/chainer_tests/links_tests/normalization_tests/test_batch_normalization.py

```diff
@@ -504,4 +505,39 @@ def test_fail_inference(self):
             links.BatchNormalization()
 
 
+@attr.multi_gpu(2)
+class TestLazyInitializationWithNonZeroCurrentCudaDevice(unittest.TestCase):
+
+    def test_lazy_initialization_with_non_zero_current_cuda_device(self):
+        x = numpy.random.randn(5, 4, 3, 2).astype('f')
```

Comment:
Dtype char code should be avoided (discouraged in numpy documentation).

Reply:
Thanks, fixed.